### DISCUSSION TITLE
Dask Server Error: Failed to list clusters

### DISCUSSION THREAD

**Discussion by sijopkd** [dask/dask#7411] [Q&A]
![Screen Shot 2021-03-16 at 2 53 23 PM](https://user-images.githubusercontent.com/24625433/111429422-f7902e00-86c6-11eb-8f03-66584274628b.png)
![Screen Shot 2021-03-15 at 11 19 45 AM](https://user-images.githubusercontent.com/24625433/111429437-fced7880-86c6-11eb-8664-ff846c136c1d.png)

I face the error on both GCP and DGX systems. After I work on the Jupyter-lab instance for around 15-20 minutes, I get these errors. After this point, I am unable to save the jupyternotebooks and the kernel stops running. Can you please help me? 

Thank you so much,
Sijo

**Comment by quasiben** [comment]
I would recommend filing an issue in [dask-labextension repo](https://github.com/dask/dask-labextension).  However, before doing that I would look through https://github.com/dask/dask-labextension/issues/87 and see if information there resolves your issue
